Cargo.toml

```diff
@@ -28,6 +28,7 @@ strum_macros = "0.26"
 
 [patch.crates-io]
 pdf-extract = { git = "https://github.com/jrmuizel/pdf-extract.git", rev = "refs/pull/91/head" }
+lax = { git = "https://github.com/mike-kfed/ndarray-linalg.git", branch = "arm-cross-compile" }
```

Comment:
Don't mind this, I just wanted builds on my laptop - will revert before publishing